content/applications/hr/fleet/service.rst

```diff
@@ -86,61 +296,219 @@ clicked, sort the data in different ways.
 
 .. image:: service/views.png
    :align: center
-   :alt: The icons in the top right corner than cn be clicked to present the information in
+   :alt: The icons in the top right corner than can be clicked to present the information in
          different ways.
 
 List view
 ~~~~~~~~~
 
-The default view of the services is the list view. This presents all the services in chronological
-order, from oldest to newest, in a list.
+The default view of the service records is a list view. This presents all the services, first
+grouped alphabetically by type of service, then grouped by status.
+
+The information can be re-sorted by any column. At the top of each column, hover over the
+:guilabel:`column name`, and an arrow appears in the far-right of that column. Click the arrow to
```

Comment:
Since there's no literal "column name" that's displayed on the UI
```suggestion
column name, and an arrow appears in the far-right of that column. Click the arrow to
```